ratatui/Cargo.toml

```diff
@@ -58,7 +58,7 @@ all-widgets = ["widget-calendar"]
 #! Widgets that add dependencies are gated behind feature flags to prevent unused transitive
 #! dependencies. The available features are:
 ## enables the [`calendar`](widgets::calendar) widget module and adds a dependency on [`time`].
-widget-calendar = ["ratatui-widgets/calendar"]
+widget-calendar = ["ratatui-widgets/calendar", "dep:time"]
```

Comment:
Side note: what about making this included in default for the widgets lib, but exclude it from the default in ratatui?
(Or even include it in the default in Ratatui as well, with the expectation that people that want smaller deps will generally disable defaults)

Reply:
Yeah, good call. #1521